core-it-suite/src/test/java/org/apache/maven/it/MavenITmng5760ResumeFeatureTest.java

```diff
@@ -40,19 +40,26 @@
  * @author Martin Kanters
  */
 public class MavenITmng5760ResumeFeatureTest extends AbstractMavenIntegrationTestCase {
-    private final File testDir;
+    private final File parentDependentTestDir;
+    private final File parentIndependentTestDir;
+    private final File noProjectTestDir;
 
     public MavenITmng5760ResumeFeatureTest() throws IOException {
         super( "[3.7.0,)" );
-        this.testDir = ResourceExtractor.simpleExtractResources( getClass(), "/mng-5760-resume-feature" );
+        this.parentDependentTestDir = ResourceExtractor.simpleExtractResources( getClass(),
+                "/mng-5760-resume-feature/parent-dependent" );
+        this.parentIndependentTestDir = ResourceExtractor.simpleExtractResources( getClass(),
+                "/mng-5760-resume-feature/parent-independent" );
+        this.noProjectTestDir = ResourceExtractor.simpleExtractResources( getClass(),
+                "/mng-5760-resume-feature/no-project" );
```

Comment:
I think we are missing these files.
After squashing and merging this branch to Apache, the GitHub Actions pipeline was failing:
https://github.com/apache/maven/runs/872356902?check_suite_focus=true
`testShouldSkipSuccessfulProjects (java.lang.IllegalArgumentException: Resource not found: /mng-5760-resume-feature/no-project`